packages/components/src/utilities/focusStyle.ts

```diff
@@ -1,13 +1,4 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { cn } from './styleUtilities';
-
-export const FOCUS_STYLE = 'tw-ring-4 tw-ring-blue tw-ring-offset-2 dark:tw-ring-offset-black tw-outline-none';
-export const FOCUS_STYLE_NO_OFFSET = 'tw-ring-4 tw-ring-blue dark:tw-ring-offset-black tw-outline-none';
-export const FOCUS_VISIBLE_STYLE =
-    'focus-visible:tw-ring-4 focus-visible:tw-ring-blue focus-visible:tw-ring-offset-2 focus-visible:dark:tw-ring-offset-black focus-visible:tw-outline-none';
-export const FOCUS_STYLE_INSET = cn([FOCUS_STYLE, 'tw-ring-inset']);
-export const FOCUS_VISIBLE_STYLE_INSET = cn([FOCUS_VISIBLE_STYLE, 'tw-ring-inset']);
-export const FOCUS_WITHIN_STYLE =
-    'focus-within:tw-ring-4 focus-within:tw-ring-blue focus-within:tw-ring-offset-2 focus-within:dark:tw-ring-offset-black focus-within:tw-outline-none';
-export const FOCUS_OUTLINE = 'focus-visible:tw-outline tw-outline-4 tw-outline-offset-2 tw-outline-blue';
+export const FOCUS_OUTLINE =
+    'focus-visible:tw-outline has-[[data-show-focus-ring="true"]]:tw-outline tw-outline-4 tw-outline-offset-2 tw-outline-blue';
```

Comment:
```suggestion
    'focus-visible:tw-rounded-[inherit] focus-visible:tw-outline has-[[data-show-focus-ring="true"]]:tw-outline tw-outline-4 tw-outline-offset-2 tw-outline-blue';
```

Reply:
fixed

Reply:
Without `inherit` it looks like this
<img width="126" alt="Screenshot 2024-05-29 at 21 55 53" src="https://github.com/Frontify/fondue/assets/6846950/05359d25-6ddd-4a8c-b072-fc4c2f1acf55">
Instead of before (notice the right side radius)
<img width="118" alt="Screenshot 2024-05-29 at 21 56 55" src="https://github.com/Frontify/fondue/assets/6846950/8adc967f-f6af-4711-866b-0ca8265c968a">

Reply:
The segmented controls is a bit of an edge case for this i believe as it is the only one of our components where the focused element has a parent with a rounding.

Other elements

Before:
<img width="177" alt="Screenshot 2024-05-30 at 08 40 49" src="https://github.com/Frontify/fondue/assets/13820687/a66c7858-85e2-47b3-9db9-08f9faf03210">
<img width="236" alt="Screenshot 2024-05-30 at 08 39 20" src="https://github.com/Frontify/fondue/assets/13820687/61006726-dc35-4208-abc1-b9ccbfb52218">

With rounded inherit:
<img width="276" alt="Screenshot 2024-05-30 at 08 41 04" src="https://github.com/Frontify/fondue/assets/13820687/9e86bdd4-c23e-4f74-9a39-169a83d6ae4e">
<img width="229" alt="Screenshot 2024-05-30 at 08 40 59" src="https://github.com/Frontify/fondue/assets/13820687/e0654339-9100-4744-be37-64a665c7a814">

Reply:
also i dislike that it doesn't just affect the outline, but also the element itself. (The button losses its rounding)

<img width="163" alt="Screenshot 2024-05-30 at 08 44 32" src="https://github.com/Frontify/fondue/assets/13820687/9f6fae5b-e699-459b-bece-735d68a7a63f">